openCurrents/views.py

```
@@ -1180,6 +1203,14 @@ def get_context_data(self, **kwargs):
         except KeyError:
             pass
 
+        # getting all admins for organization
+        context['org_admins'] = org_admins = []
+        try:
+            org_admins = [u for u in OrgUser.objects.filter(org = self.org) if OcAuth(u.id).is_admin_org]
+        except:
```

can we be explicit about which exception we are aiming to catch here

---

@nickolashe it seems that this piece of code fails silently, if it receives wrong input, let's discuss it tonight.

---

I'll need your help with this one @nickolashe 
lets discuss it during one of our conversations. Thanks!